packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.routing.js

```diff
@@ -9,7 +9,12 @@ export default /* @ngInject */ ($stateProvider) => {
         },
       },
       resolve: {
-        goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+        angularQr: /* @ngInject */ ($ocLazyLoad) =>
+          import('angular-qr').then((module) =>
+            $ocLazyLoad.inject(module.default || module),
+          ),
+        goBack: /* @ngInject */ ($state) => (reload) =>
+          $state.go('^', {}, { reload }),
```

Comment:
Are you sure it's that simple? Because if you have a success message with a reload it won't appear unless it's in the promised form, I think.